python/doc/source/other_systems.rst

```diff
@@ -0,0 +1,182 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+.. currentmodule:: pyarrow
+.. _other_systems:
+
+Using Arrow with other systems
+==============================
+
+Timestamps
+----------
+
+Timestamps are data structures that mark a particular point in time
```

Comment:
I think it's not true. In SQL standard, timestamp(by default it's TIMESTAMP WITHOUT TIMEZONE) means a "floating" time, which is kind of the seconds from local epoch, e.g. use 0 to represent "1970-1-1 00:00:00" no matter which timezone you are. In Spark SQL and Parquet, the timestamp is seconds from Unix epoch, which is a particular point in time.

Reply:
It seems like parquet has a `FloatingTimestamp` type now.